packages/admin/admin/src/tabs/RouterTabs.tsx

```diff
@@ -165,17 +165,13 @@ export function RouterTabs(inProps: Props) {
                 return (
                     <Route path={path}>
                         {({ match }) => {
-                            const ownerState: OwnerState = {
-                                contentHidden: !(match && !foundFirstMatch) && child.props.forceRender,
-                            };
                             let ret = null;
-                            if ((match && !foundFirstMatch) || child.props.forceRender) {
+
+                            if (match && !foundFirstMatch) {
                                 foundFirstMatch = true;
-                                ret = (
-                                    <Content ownerState={ownerState} {...slotProps?.content}>
-                                        {child.props.children}
-                                    </Content>
-                                );
+                                ret = <Content ownerState={{ contentHidden: false }}>{child.props.children}</Content>;
+                            } else if (child.props.forceRender) {
+                                ret = <Content ownerState={{ contentHidden: true }}>{child.props.children}</Content>;
```

Comment:
Did you remove the slotProps on purpose?

Reply:
Ah, good catch, no 😅